doc/cephadm/adoption.rst

```diff
@@ -95,6 +95,7 @@ Adoption process
      # ceph orch ps
 
 #. Adopt all OSDs in the cluster::
+   This needs to be done on ALL the OSD's in the cluster before proceeding with the steps below to adopt mds
```

Comment:
this broke the formatting. See

https://docs.ceph.com/ceph-prs/35335/cephadm/adoption/#adoption-process

Reply:
I think this is still broken. unfortunately.